pkg/services/accesscontrol/database/resource_permissions.go

```diff
@@ -614,9 +614,9 @@ func (s *AccessControlStore) getResourcePermissionsByIds(sess *sqlstore.DBSessio
 	return result, nil
 }
 
-func managedPermission(action, resource string, resourceID string) accesscontrol.Permission {
+func managedPermission(action, resource string, resourceID, resourceAttribute string) accesscontrol.Permission {
```

Comment:
nit: here and elsewhere in this file - I think we should rename `resourceID` argument to something else. For instance, `attributeValue` or `resourceAttributeValue`. Because it might not be resource ID if `resourceAttribute` isn't `id`, right?

Reply:
I think resourceID is fine, it is just and identifier for a resource and could be any type of identifier (e.g. id, uid)

Reply:
I see your point. I guess I've gotten used to interpreting ID as the ID stored in DB, so I still find it a bit confusing. But not a blocker from my side.